avgerou/roster.json

```diff
@@ -15,7 +15,8 @@
       "github": "diegorramos84",
       "materials": [
         {
-          "type": "cv"
+          "type": "cv",
+          "url": "M7E4e24V9hQ"
```

Comment:
You need to create another object, your code should look like this 😄 
```
...
{
          "type": "cv",
},
{
          "type": "video",
          "url": "M7E4e24V9hQ"
},
...
```

Reply:
yeah, obviously :P